### PR TITLE
Create test.tweet

### PR DESCRIPTION
**Pull request by RRomoff**
more tests.

### CODE REVIEW

File: tweets/test.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Second test
```
Comment (bnb) [inline tweets/test.tweet:1]:
```suggestion
First test
```

This is the first test since the other PR wasn't merged. Is this okay?

Comment (RRomoff) [reply tweets/test.tweet:1]: Sounds good!

File: tweets/test.tweet [hunk 2]
```diff
@@ -0,0 +1 @@
+Third test
```
Comment (RRomoff) [inline tweets/test.tweet:1]:
```suggestion
First tweet
```

Comment (bnb) [reply tweets/test.tweet:1]: Yep this sounds right

**Pull request closed without merging**